components/Footer/Footer.js

```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
+
+const FooterContainer = styled.footer`
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100vw;
+  min-height: 10vh;
+  background-color: white;
+  margin: 0;
+  padding: 1rem;
+  box-shadow: 0.1rem 0.4rem 0.5rem 0.55rem #dddddd;
+`;
+
+const NavList = styled.ul`
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+`;
+
+const NavItem = styled.li`
+  margin: 0 10px;
+`;
+
+const NavLink = styled.a`
```

Comment:
you should use the Link component provided by Next.js:

```suggestion
const NavLink = styled(Link)`
```

don't forget to import the component.